File: app/src/main/java/ru/santaev/clipboardtranslator/ui/MainActivityViewPagerAdapter.java
package ru.santaev.clipboardtranslator.ui;


import android.support.v4.app.Fragment;
import android.support.v4.app.FragmentManager;
import android.support.v4.app.FragmentPagerAdapter;

public class MainActivityViewPagerAdapter extends FragmentPagerAdapter{

    MainActivityViewPagerAdapter(FragmentManager fm) {
        super(fm);
    }

    @Override
    public Fragment getItem(int position) {
        switch (position){
            case 0:
                return TranslateFragment.Companion.newInstance();
            case 1:
                return HistoryFragment.newInstance();
            default:
                throw new IllegalArgumentException("Not found item for position " + position);
        }

    }

    @Override
    public int getCount() {
        return 2;
    }
}

File: app/src/main/java/ru/santaev/clipboardtranslator/ui/MainActivityViewPagerAdapter.kt
package ru.santaev.clipboardtranslator.ui


import androidx.fragment.app.Fragment
import androidx.fragment.app.FragmentManager
import androidx.fragment.app.FragmentPagerAdapter

class MainActivityViewPagerAdapter(fm: FragmentManager) : FragmentPagerAdapter(fm) {

    override fun getItem(position: Int): Fragment {
        return when (position) {
            0 -> TranslateFragment.newInstance()
            1 -> HistoryFragment.newInstance()
            else -> throw IllegalArgumentException("Not found item for position " + position)
        }

    }

    override fun getCount(): Int {
        return 2
    }
}
